From e664d52857020397720ceb51892eeda603a5e4c5 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Fri, 24 Mar 2017 22:51:06 +0000
Subject: [PATCH 08/10] rescue: docs: It is no longer necessary to mount
 filesystems by hand.

Fix the manual page to reflect the new -i option.

Fixes commit 33d2ae796119ae5dd38e2afcbf1ba4216bd99846.

(cherry picked from commit c38b48409e067ba973d02bb10273350aa31b558e)
---
 rescue/virt-rescue.pod | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/rescue/virt-rescue.pod b/rescue/virt-rescue.pod
index bd6f954e9..dfa74e204 100644
--- a/rescue/virt-rescue.pod
+++ b/rescue/virt-rescue.pod
@@ -35,9 +35,9 @@ For live VMs you I<must> use the I<--ro> option.
 When you run virt-rescue on a virtual machine or disk image, you are
 placed in an interactive bash shell where you can use many ordinary
 Linux commands.  What you see in F</> (F</bin>, F</lib> etc) is the
-rescue appliance.  You must mount the virtual machine's filesystems by
-hand.  There is an empty directory called F</sysroot> where you can
-mount filesystems.
+rescue appliance.  You must mount the virtual machine's filesystems.
+There is an empty directory called F</sysroot> where you can mount
+filesystems.
 
 To automatically mount the virtual machine's filesystems under
 F</sysroot> use the I<-i> option.  This uses libguestfs inspection to
-- 
2.13.0

